elastio-s3-changelog/cloudformation-single-bucket.yaml

```
@@ -202,6 +202,26 @@ Resources:
       Tags:
         'elastio:resource': 'true'
 
+  queueRefreshSchedule:
+    Type: AWS::Scheduler::Schedule
+    Properties:
+      Description: A schedule to invoke the periodic changelog queue refresh lambda
+      ScheduleExpression: cron(0/15 * * * ? *)
+      FlexibleTimeWindow:
+        Mode: 'OFF'
+      State: ENABLED
+      Target:
+        Arn: !Sub 'arn:aws:lambda:${AWS::Region}:${AWS::AccountId}:function:elastio-s3-changelog-handler-refresh'
+        RoleArn: !Sub 'arn:aws:iam::${AWS::AccountId}:role/ElastioS3ChangelogSchedulerExecution'
+        Input: !Sub >
+          {
+            "queue_url": "${queue}",
```

Shouldn't it be this instead?
```suggestion
            "queue_url": "${queue.QueueUrl}",
```

---

[Referring to a queue returns its url.](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-sqs-queue.html#aws-resource-sqs-queue-return-values) They also do this in examples.